src/main/frontend/pipeline-graph-view/pipeline-graph/main/support/StatusIcons.tsx

```diff
@@ -10,13 +9,10 @@ export function getGroupForResult(
   result: Result,
   percentage: number,
   radius: number
-): React.ReactElement<SvgSpinner> | React.ReactElement<SvgStatus> {
+): React.ReactElement<SvgStatus> {
```

Comment:
The Jenkins build icons use the previous result with a spinner, this now uses a similar approach for current result.